modules/virt-about-services.adoc

```diff
@@ -17,5 +17,5 @@ LoadBalancer:: Creates an external load balancer in the current cloud (if suppor
 
 [NOTE]
 ====
-For on-premise clusters, you can configure a load balancing service by using the MetalLB Operator in layer 2 mode. The BGP mode is not supported. The MetalLB Operator is installed in the `metallb-system` namespace.
+For on-premise clusters, you can configure a load-balancing service by using the MetalLB Operator. The MetalLB Operator is installed in the `metallb-system` namespace.
```

Comment:
```suggestion
For on-premise clusters, you can configure a load-balancing service by using the MetalLB Operator.
```

Saying where the operator is installed is oddly specific. Could we use this opportunity to drop it? Otherwise LGTM.